modules/exploits/linux/http/ibm_drm_rce.rb

```diff
@@ -0,0 +1,281 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
```

Comment:
While we're waiting for CVEs, can you change this URL to `https://`? Same for #13301. Travis is yelling at us for that, plus the `TODO` CVEs.